src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -231,6 +235,11 @@ private static boolean shouldPreferVideoHearingChannel(HearingSubtype hearingSub
     }
 
     public static String getIndividualInterpreterLanguage(HearingOptions hearingOptions, ReferenceDataServiceHolder referenceData) throws InvalidMappingException {
+
+        if (isNull(hearingOptions)) {
+            return "";
```

Comment:
Can we add a empty string or reuse one from AppConstants (if we already have one) ?